applications/skyepass.md

```diff
@@ -0,0 +1,251 @@
+* Zoe Sun (Lead Designer)
+* ... More to be hired 
+
+### Contact
+* **Contact Name:** Song Zhou (song.zhou@ponder.capital)
+
+### Legal Structure 
+* No legal entity yet
+
+### Team's experience
+Besides private work for companies that cannot be shared, Song developed a simple server-less React.js Blog system(can be seen on his Github profile); a private event participation checkin application, based on Ethereum smart contract, React.js for frontend, Coda.io API and a Telegram bot for administration. 
+
+### Team Code Repos
+* https://github.com/skyekiwi/skyepass
+
+## Development Roadmap :nut_and_bolt: 
+
+### Overview
+* **Total Estimated Duration:** 16 weeks
+* **Full-time equivalent (FTE):**  2.5 FTE 
+* **Total Costs:** See Below
+
+### Milestone 1 — Core Encryption/Identity Management Smart Conctracts/Basic Desktop App/Basic Browser Extension 
+* **Estimated Duration:** 10 Weeks
+
+* **FTE:**  2
+
+* **Costs:** 26,600 DAI (2 FTE * 35 Hours per week * 10 Weeks * $38 Hourly Wage. Of course, I'll be surprised if we will actually work less than 50 hours per week.) 
+
+    
+
+| Number | Deliverable                    | Specification                                                |
+| ------ | ------------------------------ | ------------------------------------------------------------ |
+| 0a.    | License                        | Apache 2.0                                                   |
+| 0b.    | Documentations/Project Website | A minimalism project homepage with a user manual that documents how to use SkyePass from account creation to manage passwords, and install extensions and browser extensions. <br/>A developer documentation that describes the encryption mechanism in details; Basic API documentation for extension development. |
+| 1.     | Desktop App/Browser Extension  | **Desktop App**<br/>A React.js + Electron App to start with. The App will implement as close as possible to the graphic design (per 0d). <br/>- Wallet Creation / Backup Phase / Create Master Password<br/>- Wallet Import / Signin<br/>- Autolock after timed inactivity or manually lock the App <br/>- Create/Share/Manage Vaults<br/>- Add/Update/Delete Password Items (with 2FA OTP support)<br/>- Add/Update/Delete Secure Note/Credit Card <br/>- Basic ETH wallet extension<br/>- Basic Polkadot wallet extension <br/>- Application Marketplace<br/>- link social media or other identity handle to a public account<br/>- sharing a single password item directly to another user<br/><br/>**Browser Extension** <br/>- Communication to Desktop Application<br/>- Auto-fill account/passwords<br/> <br/>**IPFS Connection**<br/>- upload / update / read from a public IPFS node. |
+| 2.     | Smart Contract                 | All 4 smart contracts mentioned in the `Ecosystem Fit` section. For this phase, these contracts are testing contracts, and they are subject to minor changes as long as they provide the same/very similar functionality. |
```

Comment:
Which programming languages are you planning to use here? We mostly want to support the development of new languages like [ink!](https://substrate.dev/docs/en/knowledgebase/smart-contracts/).

Reply:
Yes, Rust, ink! and tools by [Patract.io](https://patract.io/?lang=en-US) starting with Jupiter. JS for client side, of course. Polkadot.js for interacting with the blockchain.

Reply:
Nice. Could you integrate this into the deliverables?

Reply:
Done.